SOPT-iOS/Projects/Domain/Sources/UseCase/ShowAttendanceUseCase.swift

```diff
@@ -75,7 +86,28 @@ extension DefaultShowAttendanceUseCase: ShowAttendanceUseCase {
             .store(in: cancelBag)
     }
     
-    /// [1차 출석 , 2차 출석, 출석] 각각 상태 및 타이틀 가지는 배열
+    public func fetchLectureRound(lectureId: Int) {
+        repository.fetchLectureRound(lectureId: lectureId)
+            .catch({ error in
+                if let error = error as? OPAPIError {
+                    var errorMsg = error.errorDescription ?? ""
+                    self.setLectureRoundErrorTitle(&errorMsg)
+                    self.lectureRoundErrorTitle.send(errorMsg)
+                }
+                
+                return Just<AttendanceRoundModel?>(nil)
+            })
```

Comment:
위에 코멘트 남긴 부분 반영되면 호출부는 이렇게 될거같아여
```suggestion
.catch({ error in
            if let errorMsg = AttendanceErrorMsgType.getTitle(for: error) {
                self.lectureRoundErrorTitle.send(errorMsg)
            }
            return Just<AttendanceRoundModel?>(nil)
        })
```

Reply:
조아요 ~~! 👍🏻